src/index.html

```diff
@@ -11,37 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <form method="get" data-qa="big">
-      <label class="label">
-        <input
-          data-qa="keypress"
-          class="form-field" 
-          type="text" 
-          name="search" 
-          placeholder="Try &quot;Los Angeles&quot;"
-        >
-        <img 
-          src="images/Search.svg" 
-          alt="bigSearch" 
-          class="big-zoom"
-        >
-      </label>
+    <form 
+      method="get" 
+      data-qa="big"
+      class="form"
+    >
+      <label for="big"></label>
```

Comment:
delete `label`, you don't need it

Reply:
"Input with id "null" has no associated label" - when i delete label

Reply:
so delete id from input also